src/pages/Home/Home.tsx

```diff
@@ -1,49 +1,57 @@
 // We will need to replace with our actualy products
-import { useNavigate } from "react-router-dom";
-import { products } from "../../data";
-import { useCart } from "../../ShoppingCartContext";
+import { useNavigate } from 'react-router-dom';
+import { products } from '../../data';
+import { useCart } from '../../ShoppingCartContext';
 
-import PlusMinusButton from "../../components/PlusMinusButton/PlusMinusButton";
-import Button from "../../components/Button/Button";
+import PlusMinusButton from '../../components/PlusMinusButton/PlusMinusButton';
+import Button from '../../components/Button/Button';
+import ProductCard from '../../components/ProductCard/ProductCard';
 
 function Home() {
-  const navigate = useNavigate();
+	const navigate = useNavigate();
 
-  const { addToCart } = useCart();
+	const { addToCart } = useCart();
 
-  function handleAddToCart(productId: number) {
-    const product = products.find((p) => p.id === productId);
-    if (product) {
-      addToCart({ ...product, quantity: 1 });
-    }
-  }
+	function handleAddToCart(productId: number) {
+		const product = products.find((p) => p.id === productId);
+		if (product) {
+			addToCart({ ...product, quantity: 1 });
+		}
+	}
 
-  //Move to nav bar component!!
-  function goToShoppingCart() {
-    navigate("/shopping-cart");
-  }
+	//Move to nav bar component!!
+	function goToShoppingCart() {
+		navigate('/shopping-cart');
+	}
 
-  return (
-    <div>
-      <h1>Products</h1>
-      {/* Put in nav bar component!! */}
-      <Button
-        buttonText="Shopping Cart"
-        buttonClick={() => goToShoppingCart()}
-        size="large"
-      />
-      <ul>
-        {products.map((product) => (
-          <li key={product.id}>
-            {product.name} - £{product.price}
-            <PlusMinusButton
-              buttonText="+"
-              buttonClick={() => handleAddToCart(product.id)}
-            />
-          </li>
-        ))}
-      </ul>
-    </div>
-  );
+	return (
+		<div>
+			<h1>Products</h1>
+			{/* Put in nav bar component!! */}
+			<Button
+				buttonText='Shopping Cart'
+				buttonClick={() => goToShoppingCart()}
+				size='large'
+			/>
+			<ul>
+				{products.map((product) => (
+					<ProductCard
```

Comment:
React is a little unhappy with this. I get this error: ``` Each child in a list should have a unique "key" prop.```. 

I believe it can be addressed by adding ```key={product.id}``` to each product card like so ```<ProductCard key={product.id} />```

Reply:
Ah thanks for that - I'll add it :)